compiler/rustc_typeck/src/collect.rs

```diff
@@ -2224,6 +2219,8 @@ fn gather_explicit_predicates_of(tcx: TyCtxt<'_>, def_id: DefId) -> ty::GenericP
         + has_own_self as u32
         + early_bound_lifetimes_from_generics(tcx, ast_generics).count() as u32;
 
+    debug!("gather_explicit_predicates_of(predicates={:?})", predicates);
+    debug!("gather_explicit_predicates_of(ast_generics={:?})", ast_generics);
```

Comment:
if you're using `instrument`, don't mention the function name in the tracing statements again
```suggestion
    debug!(?predicates);
    debug!(?ast_generics);
```